deploy/docker-ephemeral/build/README.md

```diff
@@ -0,0 +1,54 @@
+A makefile that uses docker.io, and qemu-user-static to build dependencies for our integration tests.
+
+Builds and uploadsdocker images for multiple architectures. Allows for '-j' to build multiple images at once. Uploads assume the hub.docker.com docker registry.
+
+# Setup
+
+## Docker
+
+Follow the instructions in [our dependencies file](doc/Dependencies.md) to ensure you have docker installed, and logged in.
+
+## qemu
+
+### Debian
+
+
+```bash
+apt-get install qemu-user-static
+sudo service binfmt-support start
+```
+
+### Fedora
+
+'sudo dnf install -y qemu-user-static'
+
+# Using
+
+Assuming you have docker, and have followed the above instructions, "make build-all" should work. This builds all of the images, and places them in docker on the local machine.
+to build an individual image (and it's dependent images), run "make-<imagename>". to see a list of images that are buildable, run "make names".
+
+## Using with Dockerhub
+
+If you want to upload images to dockerhub, you must go to dockerhub, and create repositories under your user with the names of the images you want to upload. Again, to get the list of names buildable with this Makefile, type 'make names'.
```

Comment:
The `Again,` seems maybe a bit sassy 😼 ; I typically wouldn't assume people read README's from start to finish; probably nicer to just state the information plainly again 😄 
```suggestion
If you want to upload images to dockerhub, you must go to dockerhub, and create repositories under your user with the names of the images you want to upload. Use `make names` to get the list of built images.
```